test/addressfield-test.js

```diff
@@ -336,6 +336,37 @@
     expect(17);
   });
 
+  test('select a default country', function() {
+    var $cloneAddress = this.address.clone(),
+        $emptyCountry = $cloneAddress.find('.country'),
+        mockJsonData = {
+          "label": "Country",
+          "options": [
+            {
+              "label": "United States",
+              "iso": "US"
+            },
+            {
+              "label": "Canada",
+              "iso": "CA"
+            },
+            {
+              "label": "Japan",
+              "iso": "JA"
+            }
+          ]
+        };
+
+    $emptyCountry.attr("data-country-selected", "US");
```

Comment:
Looks like [the fixture defaults the select value](https://github.com/tableau-mkt/jquery.addressfield/blob/master/test/addressfield.html#L31) to `US` already.  I think the test would be more meaningful if you defaulted to another country.